Gemfile

```diff
@@ -1,7 +1,7 @@
 source "http://rubygems.org"
 
 gem 'resque', '~>1.21'
-gem "ffi-rzmq", "~>1.0"
+gem "ffi-rzmq", "~>2.0"
```

Comment:
Yes, that seems the likely reason. They should include it if they need it themselves.